Library/Formula/xalan-c.rb

```diff
@@ -0,0 +1,58 @@
+class XalanC < Formula
+  desc "XSLT processor"
+  homepage "https://xalan.apache.org/xalan-c/"
+  url "https://www.apache.org/dyn/closer.cgi?path=xalan/xalan-c/sources/xalan_c-1.11-src.tar.gz"
+  sha256 "4f5e7f75733d72e30a2165f9fdb9371831cf6ff0d1997b1fb64cdd5dc2126a28"
+
+  option :universal
+
+  depends_on "xerces-c"
+
+  def install
+    ENV.deparallelize # This build isn't parallel safe.
```

Comment:
Can you add that in a comment about `ENV.deparallelize`, so we know if/when to remove it? Thanks!